samples/router/sites/sample.md

```diff
@@ -11,9 +11,9 @@ include: bridge-to-router-note
 ## Running the project
 
  1. Start the projects in debug mode.
- 1. Press <kbr>Enter</kbr> a couple of times.
- 1. Observe the `PingHandler` logging processed message IDs in the Server window.
- 1. Observe the `PongHandler` logging processed message IDs in the Client window.
+ 1. Press <kbd>Enter</kbd> a couple of times.
```

Comment:
"couple" is 2: is it exactly 2 times that the Enter key needs to be pressed?

Reply:
```suggestion
 1. Press <kbd>Enter</kbd> a few times.
```